src/MsGraph.php

```diff
@@ -94,13 +94,20 @@ public function connect($id = null)
                     'token_id' => $result->id,
                     'info' => $me
                 ];
+                
+                //Check if mail is null.  If so, use userPrincialName for email
+                if ($me['mail']===null) {
```

Comment:
please add formatting:

```
if ($me['mail'] === null) {
    $email = $me['userPrincipalName'];
} else {
    $email = $me['mail'];
}
```

Reply:
Something isn't working properly after I merged in the current version.  I need to do some debugging so need to abandon this request for now